andela/ah-92explorers-backend
# #166954578 Fix Failing Build on Travis-CI

#### What does this PR do?
Fixes failing build on travisci

#### Description of Task to be completed
- remove tests that are connected to mongodb
- add a test for the `/welcome` endpoint.
- add npm test in .travis.yml to fix the failing build on travis-ci

#### How should this be manually tested?

1. Run the following commands in Terminal

```!#bash
git clone git@github.com:andela/ah-92explorers-backend.git
cd ah-92explorers-backend
git checkout bg-fix-failing-build-on-travisci-166954578
npm run test
```

2. Navigate to `.travis.yml` file in the root directory. You will see the defaults settings

3. The `README.md` file also has a Travis CI badge to show the build status of the project.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#166954578](https://www.pivotaltracker.com/story/show/166954578)

#### Checklist
